Reference/Searching/Examine/Quick-Start/index.md

```diff
@@ -25,39 +23,78 @@ We will make it possible to 'search' on the _People_ page, by adding a search ba
 -->
 <div>
     <form action="@Model.Url()" method="get">
-        <input type="text" placeholder="Search" id="query" name="query" />
+        <input type="text" placeholder="Search" name="query"/>
         <button>Search</button>
     </form>
 </div>
 <div class="employee-grid">
 ...
 ```
 This will create a basic input field at the top of the page and make it post to the same people page when submitted along with the search term.
-### Retrieving the Search Term
+### Using route hijacking
 
-Right below the form, add the following:
+Now that we can post to the people view, we want a bit more control, we probably want a custom ViewModel, with all the search results, when people try to search for something!
+We will use route hijacking by creating a render controller, for more information about route hijacking and render controllers, you can read our documentation: https://our.umbraco.com/documentation/reference/routing/custom-controllers/
+
+Lets start by creating a PeopleController, and letting it derive from `RenderController`, and then adding an Index method.
+:::note Its important to name our Controller by the convention `NameOfViewController` so in our case our view is named People, so we call it PeopleController
```

Comment:
Suggest new wording - something in the lines of: 

The best practice for POST requests is to encapsulate the request handling in a controller. To do this we will leverage the concept of [route hijacking](https://our.umbraco.com/documentation/reference/routing/custom-controllers/). 

Lets start by creating a `PeopleController` that derives from `RenderController` and add an `Index` method. 

:::note 
It is important to name our controller by the convention `_NameOfViewController_`. In our case the view is named People, so the controller is named `PeopleController`.
:::